fem/fespace.hpp

```diff
@@ -712,6 +844,20 @@ class FiniteElementSpace
       return dynamic_cast<const L2_FECollection*>(fec) != NULL;
    }
 
+   /** In variable order spaces on nonconforming meshes, this function controls
+       whether strict conformity is enforced in cases where master edges/faces
+       have higher polynomial order than their refined slaves. In the default
+       (strict) case, the master polynomial order is reduced to that of the
+       lowest order slave, so the slave can interpolate the master side exactly.
+       If relaxed == true, discontinuities in the solution in such cases are
+       allowed and the master side is not restricted. */
+   void SetRelaxedHpConformity(bool relaxed = true)
+   {
+      relaxed_hp = relaxed;
+      orders_changed = true; // force update
```

Comment:
Wouldn't `needs_update` be more clear than `orders_changed`?

Reply:
No, `needs_update` would be too broad. This means exactly what it says, `orders_changed` is `true` when any of the element order changed. There may be other reasons the space needs updating (e.g. the mesh changed).